### PR TITLE
Fixes #8974: Install bash autocompletion from the package

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8974

### CODE REVIEW

File: rudder-agent/SPECS/rudder-agent.spec [hunk 1]
```diff
@@ -371,6 +371,7 @@ mkdir -p %{buildroot}%{ruddervardir}/tmp
 mkdir -p %{buildroot}%{ruddervardir}/tools
 mkdir -p %{buildroot}%{rudderlogdir}/install
 mkdir -p %{buildroot}%{bindir}
+mkdir -p %{buildroot}/etc/bash_completion.d
```
Comment (amousset) [inline rudder-agent/SPECS/rudder-agent.spec]: Is this relevant on AIX?

Comment (peckpeck) [inline rudder-agent/SPECS/rudder-agent.spec]: No

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**